wordpress-mobile/gutenberg-mobile
# Allow writing custom HTML in HTML mode

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/2292

This PR registers the classic block for freeform support when parsing a post. It does not provide an Edit feature for the classic block though

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/22609

### Testing Instructions
- Boot WPAndroid
- Switch to HTML mode
- Type text
- Wait for a second
- Notice text is not replaced by `<p>text</p>`
- Switch back to visual
- Notice unsupported block is displayed


### Warning
- If a post with only HTML (no blocks) is saved it will load with aztec whenever it's opened again
- Switching back and forth between visual and code mode might transform the HTML into paragraphs (or other blocks) as the web does

PR submission checklist:
